next.config.js

```diff
@@ -30,17 +30,28 @@ const nextConfig = withCSS({
     // eslint-disable-next-line no-param-reassign
     config.node = { fs: 'empty' };
 
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: [
-        {
-          loader: 'react-svg-loader',
-          options: {
-            svgo: svgoConfig,
+    config.module.rules.push(
+      {
+        test: /\.svg$/,
+        use: [
+          {
+            loader: 'react-svg-loader',
+            options: {
+              svgo: svgoConfig,
+            },
           },
-        },
-      ],
-    });
+        ],
+      },
+      {
+        test: /\.(txt|png|jpg|gif)$/i,
```

Comment:
Ah this is why the Next export isn't working...

https://github.com/zeit/next.js/issues/3586#issuecomment-365253390

```suggestion
        test: /\.(jpe?g|png|gif|txt)$/,
        use: [
          {
            loader: "url-loader",
            options: {
              limit: 8192,
              fallback: "file-loader",
              publicPath: "/_next/",
              outputPath: "static/images/",
              name: "[name]-[hash].[ext]"
            }
          }
        ]
      }
```